.github/CODEOWNERS

```diff
@@ -58,9 +58,11 @@
 /maintainers/scripts/db-to-md.sh @jtojnar @ryantm
 /maintainers/scripts/doc @jtojnar @ryantm
 
-/doc/build-aux/pandoc-filters @jtojnar
-/doc/contributing/ @fricklerhandwerk
-/doc/contributing/contributing-to-documentation.chapter.md @jtojnar @fricklerhandwerk
+# Contributor documentation
+/CONTRIBUTING.md @infinisil
+/.github/PULL_REQUEST_TEMPLATE.md
```

Comment:
Oh damn yeah, sorry I merged it already, but I'll open a new one for that!

Reply:
Argh sorry, I should have been more careful. Also why does github say the file is valid with a basically empty entry?

Reply:
I guess it's not technically invalid, it just doesn't make much sense, I wouldn't mind them adding a check for that. It does have a slight effect though because codeowners of subdirectories override code owners of the parents.

Reply:
Because it actually is valid: A line with no owner is used to exclude a path from ownership. 
```
# In this example, @octocat owns any file in the `/apps`
# directory in the root of your repository except for the `/apps/github`
# subdirectory, as its owners are left empty.
/apps/ @octocat
/apps/github
```
- [Codeowners docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/248168